src/picongpu/include/compileTime/conversion/TypeToPointerPair.hpp

```diff
@@ -49,6 +49,12 @@ struct MakeIdentifier
     typedef TypeAsIdentifier<T_Type> type;
 };
 
+template<typename T_Type>
```

Comment:
/** Pass through of an already existing Identifier
 *
 * Avoids double-wrapping of an Identifier
 */